### PR TITLE
Add "new rejudging" page.

### PR DESCRIPTION
**Pull request by meisterT**

### CODE REVIEW

File: webapp/src/DOMJudgeBundle/Controller/Jury/RejudgingController.php [hunk 1]
```diff
@@ -419,6 +420,82 @@ public function finishAction(Request $request, RejudgingService $rejudgingServic
         }
     }
 
+    /**
+     * @Route("/rejudging/add", name="jury_rejudging_add")
```
Comment (nickygerritsen) [inline webapp/src/DOMJudgeBundle/Controller/Jury/RejudgingController.php]: Nitpick: the other endpoints are under `/rejudgings` (and we have that for other things likes teams and users as well), so I'd do that here too.

File: webapp/src/DOMJudgeBundle/Controller/Jury/RejudgingController.php [hunk 2]
```diff
@@ -419,6 +420,82 @@ public function finishAction(Request $request, RejudgingService $rejudgingServic
...
+                $queryBuilder
+                    ->andWhere('j.result IN (:verdicts)')
+                    ->setParameter(':verdicts', $verdicts);
+            }
+
+            /** @var array[] $judgings */
+            $judgings = $queryBuilder
+                ->getQuery()
+                ->getResult(Query::HYDRATE_ARRAY);
+            dump($judgings);
```
Comment (nickygerritsen) [inline webapp/src/DOMJudgeBundle/Controller/Jury/RejudgingController.php]: Woops 😄

File: webapp/src/DOMJudgeBundle/Resources/views/jury/rejudging_form.html.twig [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{% extends "@DOMJudge/jury/base.html.twig" %}
+
```
Comment (nickygerritsen) [inline webapp/src/DOMJudgeBundle/Resources/views/jury/rejudging_form.html.twig]: If you add

```twig
{% import "@DOMJudge/jury/jury_macros.twig" as macros %}

{% block extrahead %}
    {{ parent() }}
    {{ macros.select2_extrahead() }}
{% endblock %}
```

you get the Select2-based multi-selects, which IMHO look nicer (and we also have in some other places like judgehost restrictions)

File: webapp/src/DOMJudgeBundle/Resources/views/jury/rejudging_form.html.twig [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+{% block title %}Add rejudging - {{ parent() }}{% endblock %}
+
+{% block content %}
+
+    <h1>Add rejudging</h1>
+
+    <div class="row">
+        <div class="col-lg-4">
+            {{ form_start(form) }}
```
Comment (nickygerritsen) [inline webapp/src/DOMJudgeBundle/Resources/views/jury/rejudging_form.html.twig]: I think you output the whole form? If so, you can replace all this by
```twig
{{ form(form) }}
```

File: webapp/src/DOMJudgeBundle/Resources/views/jury/rejudging_form.html.twig (hunk 3 above)
Comment (meisterT) [reply webapp/src/DOMJudgeBundle/Resources/views/jury/rejudging_form.html.twig]: beautiful, thanks!

File: webapp/src/DOMJudgeBundle/Resources/views/jury/rejudging_form.html.twig (hunk 4 above)
Comment (meisterT) [reply webapp/src/DOMJudgeBundle/Resources/views/jury/rejudging_form.html.twig]: That's easy :-D

**Pull request merged**